cmd/user-manager/proxy/proxy.go

```diff
@@ -0,0 +1,34 @@
+package proxy
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func SyncUsers() error {
+	proxyUser := "root"
+	proxyPass := os.Getenv("MYSQL_ROOT_PASSWORD")
+	proxyHost := os.Getenv("PROXY_SERICE")
+	proxyPort := "3306"
+	clusterUser := "root"
+	clusterPass := os.Getenv("MYSQL_ROOT_PASSWORD")
+	clusterHost := os.Getenv("PXC_SERICE")
+
+	pat := exec.Command("bash", "proxysql-admin",
```

Comment:
I think bash not needed if the file copied correctly
```suggestion
	pat := exec.Command("proxysql-admin",
```